contracts/core/VolOracle.sol

```diff
@@ -174,21 +204,79 @@ contract VolOracle is DSMath {
                 .mul(annualizationConstant);
     }
 
+    /**
+     * @notice Returns the TWAP for observation period in USDC terms
+     * @param pool is the Uniswap V3 pool for which we query a TWAP
+     */
+    function assetPriceInUSDC(address pool)
+        public
+        view
+        returns (uint256 price)
+    {
+        IUniswapV3Pool uniPool = IUniswapV3Pool(pool);
+        address token0 = uniPool.token0();
+        address token1 = uniPool.token1();
+        bool isETHPair = token0 == WETH || token1 == WETH;
+
+        if (pool == ethUsdcPool) {
+            // Since this is the ETHUSDC pool, we just use token0, which is USDC as the quoteToken
+            return _twap(pool, token0, token1, 0);
```

Comment:
we are positive token0 = usdc?

Reply:
Yep it 
![Screenshot 2021-11-03 at 2 49 20 PM](https://user-images.githubusercontent.com/5748750/140019247-834e9530-fa17-4f9b-9c03-69e87a14f8a6.png)
is